src/QMCWaveFunctions/lcao/LCAOrbitalSet.h

```diff
@@ -40,6 +43,10 @@ namespace qmcplusplus
      * makeClone makes a shallow copy
      */
     ValueMatrix_t* C;
+    ValueMatrix_t* m_init_B;
+
+    ParticleSet::ParticleGradient_t myG_temp, myG_J;
+    ParticleSet::ParticleLaplacian_t myL_temp, myL_J;
```

Comment:
These four elements do not really belong to SPO conceptually. If you really need to use them in evaluateDerivates, create it inside the routine.